src/spirc.rs

```diff
@@ -228,6 +256,27 @@ impl Future for SpircTask {
 impl SpircTask {
     fn handle_command(&mut self, cmd: SpircCommand) {
         match cmd {
+            SpircCommand::Play => {
+                CommandSender::new(self, MessageType::kMessageTypePlay).send();
```

Comment:
Broadcasting these messages isn't great, ideally we'd want to track the currently active device.
It's probably fine for now, we'll assume clients just ignore them if they aren't active.

Can you please add a TODO though.